### PR TITLE
又新增一個button

### PR DESCRIPTION
**Pull request by AzusaChen**
WTF

### CODE REVIEW

**Comment by JDW-Daniel** [comment]
WTF是怎樣

**Comment by AzusaChen** [comment]
What the fuck!

File: MSIT124Practice/MSIT124Practice/obj/Debug/MSIT124Practice.csproj.CoreCompileInputs.cache [hunk 1]
```diff
@@ -1 +1 @@
-d72c45f21beb3389ffa9ac404316aee4efc40ef3
+c4b87d5089349dd1a526bdc1486c87e6c0a19e2d
```
Comment (JDW-Daniel) [inline MSIT124Practice/MSIT124Practice/obj/Debug/MSIT124Practice.csproj.CoreCompileInputs.cache]: 這行是三小

**Comment by JDW-Daniel** [comment]
確定是這行?
https://github.com/JDW-Daniel/MSIT124Pratice/pull/1/commits/f9b688eb7ef5a9b4c2a8df7681adbf73875d327d#diff-4ada96ba61fab68f1658208926830f9aR1

**Pull request merged**